edk2toollib/uefi/edk2/parsers/base_parser.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, log):
         self.PPs = []
         self.TargetFile = None
         self.TargetFilePath = None
+        self.MacroNotDefinedValue = "0"  # value to used for undefined macro
```

Comment:
If we expect this to be somewhat private to the class, maybe we should change this to self._MacroNotDefinedValue